spec/features/company_view_spec.rb

```diff
@@ -69,68 +69,68 @@
     end
   end
 
-  #describe "#email_addresses" do
+  describe "#email_addresses" do
 
-    #before(:each) do
-      #company.email_addresses.create(address: "molsen@example.com")
-      #company.email_addresses.create(address: "belieber@example.com")
+    before(:each) do
+      company.email_addresses.create(address: "molsen@example.com")
+      company.email_addresses.create(address: "belieber@example.com")
 
-      #visit company_path(company)
-    #end
+      visit company_path(company)
+    end
 
-    #it "shows the email addresses" do
-      #company.email_addresses.each do |email|
-        #expect(page).to have_selector('li', text: "#{email.address}")
-      #end
-    #end
+    it "shows the email addresses" do
+      company.email_addresses.each do |email|
+        expect(page).to have_selector('li', text: "#{email.address}")
+      end
+    end
 
-    #it "has a link to add an email address" do
-      #expect(page).to have_link('Add email address', href: new_email_address_path(company_id: company.id))
-    #end
+    it "has a link to add an email address" do
+      expect(page).to have_link('Add email address', href: new_email_address_path(contact_id: company.id, contact_type: 'Company'))
```

Comment:
Line is too long. [131/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.